x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -172,3 +172,35 @@ export const cleanupCredentials = (packagePolicy: NewPackagePolicy | UpdatePacka
   // nothing to do, return unmutated policy
   return packagePolicy;
 };
+
+export const getBenchmarkCisName = (benchmarkId: string) => {
```

Comment:
nit: in the last status we've discussed this should be the same name as it appears on `rule.benchmark.name`